### DISCUSSION TITLE
New maintainer addition

### DISCUSSION THREAD

**Discussion by grzuy** [kickstarter/rack-attack#637] [General]
Hi @ioquatix,

Can you please give @santib "Maintainer" collaborator permissions on rack-attack repo?

I've been struggling lately to find the proper time to tackle some of the maintainer duties, so we, and all the community, can definitely benefit from more his active help on this project :smiley: 

Santiago's been contributing lately with several new PRs, and notoriously with some extremely helpful and detailed code reviews on complex long standing open PRs, for which I am super grateful.

I already talked with him directly and his is onboard with the idea.

Thanks in advance.

**Comment by ioquatix** [comment]
I sent the invite, thanks for your efforts as a maintainer of this project.

**Comment by santib** [comment]
Thank you @grzuy @ioquatix 🙌
